launch-dedicated.sh

```diff
@@ -23,7 +28,12 @@ ADVERTISE_ONLINE="${AdvertiseOnline:-"True"}"
 ENABLE_SINGLE_PLAYER="${EnableSingleplayer:-"False"}"
 PASSWORD="${Password:-""}"
 
-cd engine
+cd "${TEMPLATE_ROOT}"
+if [ ! -d "${ENGINE_DIRECTORY}" ]; then
+	make -f Makefile
```

Comment:
Why are you doing this?

make will see makefiles in the current directory and use that before the makefile in the engine.